src/web/lib/getCountryCode.ts

```diff
@@ -33,6 +33,30 @@ export const setCountryCode = (countryCode: string): void => {
     });
 };
 
+export const setCountryCodeSynchronous = (countryCode: string): void => {
+    if (countryCode) {
+        localStorage.setItem(
+            COUNTRY_CODE_KEY,
+            JSON.stringify({
+                value: countryCode,
+                expires: new Date().getTime() + TEN_DAYS,
+            }),
+        );
+    }
+};
+
+export const getCountryCodeFromLocalStorage = (): string | null => {
```

Comment:
Could you just `return` from the `try...catch` block ?

```
    try {
        const item = localStorage.getItem(COUNTRY_CODE_KEY);
        localCountryCode = item ? JSON.parse(item) : null; 
        return localCountryCode.value || null
    } catch (error) {
        return null
    }
```
   

```

Reply:
thanks. will fix before merging